packages/desktop-client/src/components/reports/spreadsheets/recalculate.tsx

```diff
@@ -0,0 +1,47 @@
+import * as d from 'date-fns';
+
+import { integerToAmount } from 'loot-core/src/shared/util';
+
+import filterHiddenItems from './filterHiddenItems';
+
+function recalculate(item, months, assets, debts, groupByLabel) {
+  let totalAssets = 0;
+  let totalDebts = 0;
+  const monthData = months.reduce((arr, month) => {
+    let monthAssets = filterHiddenItems(item, assets)
+      .filter(asset => asset.date === month && asset[groupByLabel] === item.id)
+      .reduce((a, v) => (a = a + v.amount), 0);
+    totalAssets += monthAssets;
+
+    let monthDebts = filterHiddenItems(item, debts)
+      .filter(debt => debt.date === month && debt[groupByLabel] === item.id)
+      .reduce((a, v) => (a = a + v.amount), 0);
+    totalDebts += monthDebts;
+
+    const dateParse = d.parseISO(`${month}-01`);
+    //const change = last ? total - amountToInteger(last.totalTotals) : 0;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
```